selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateNamedFactoryFromDashBoard.java

```diff
@@ -94,7 +96,14 @@ public void createFactoryFromDashBoard() throws ExecutionException, InterruptedE
     events.waitExpectedMessage(CONFIGURING_PROJECT_AND_CLONING_SOURCE_CODE);
     events.waitExpectedMessage("Project " + PROJECT_NAME + " imported");
     notificationsPopupPanel.waitPopUpPanelsIsClosed();
-    projectExplorer.openItemByPath(PROJECT_NAME);
+
+    try {
+      projectExplorer.openItemByPath(PROJECT_NAME);
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7959");
```

Comment:
It's better to pass "_ex_" variable into the **fail()** method to have additional info about the error in the test log.

Reply:
Please keep the output of the log short. I think a message is enough. No need to put giant stack trace to output.

Reply:
okay